backend/no_waste.py

```diff
@@ -0,0 +1,75 @@
+import os # biblioteka systemowa
+from flask import Flask, json, abort
+from flask_sqlalchemy import SQLAlchemy
+
+app = Flask(__name__)
+# glowny modul flask 
+
+#konfiguracja bazy danych
+basedir = os.path.abspath(os.path.dirname(__file__)) # katalog aplikacji
+# __file__ <- sciezka biezacego pliku
+# os.path.dirname <- wez katalog ze sciezki (np /a/b/c -> /a/b)
+# os.path.abspath <- rozwin sciezke np ~/a/b/c -> /home/user/a/b/c
+
+app.config['SECRET_KEY'] = 'hard to guess string'
+app.config['SQLALCHEMY_DATABASE_URI'] =\
+    'sqlite:///' + os.path.join(basedir, 'data.sqlite')
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+
+#konfiguracja bazy danych sqlAlchemy
+
+db = SQLAlchemy(app)
+
+db.create_all()
+
+# jaka baza?
+
+# przepis (na razie wersja podstawowa bez wyciagnietych skladnikow itp)
+class Recipe(db.Model):
+    __tablename__ = 'recipes'
+    id = db.Column(db.Integer, primary_key=True)
+    recipe_name = db.Column(db.String(500))
```

Comment:
I don't think that 500 is needed. 255 in most cases allows you to enabled some DB optimization.